daemon-tests/tests/main/offer.rs

```diff
@@ -34,15 +39,25 @@ async fn taker_receives_offer_from_maker_on_publication(contract_symbol: Contrac
                 .build(),
         )
         .await;
+}
 
+/// Verify that an offer with a given contract symbol gets published by the
+/// maker an is received by the taker
+async fn assert_offer(maker: &mut Maker, taker: &mut Taker, symbol: ContractSymbol) {
+    publish_offer(maker, symbol).await;
```

Comment:
❓ Isn't it a bit weird to act and assert in the same test function? I would expect the `publish` bit to happen outside. When reading the tests I was quite confused.

Reply:
I'll rename the function, as indeed `assert_` name was confusing.